jetty-ee10/jetty-ee10-servlet/src/main/java/org/eclipse/jetty/ee10/servlet/DefaultServlet.java

```diff
@@ -73,7 +74,7 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
-public class DefaultServlet extends HttpServlet
+public class DefaultServlet extends HttpServlet implements ResourceFactory
```

Comment:
Not sure this is correct.
ResourceFactory is for creating new Resources objects.
DefaultServlet needs to resolve Resources from an existing ResourceBase-ish.
Ew, the ResourceService should be handling this, and the overridden ResourceService in the DefaultServlet should be handling this ee10 specific behavior.

Reply:
The easiest way would be to combine the CSS, Favicon and base resources into a `ResourceCollection`. But this requires https://github.com/eclipse/jetty.project/pull/8435 to be merged first.